lib/modules/datasource/deb/index.spec.ts

```diff
@@ -537,7 +434,7 @@ function mockFetchInReleaseContent(
  * @param arch - The architecture name (e.g., 'amd64', 'i386').
  * @returns The complete URL to the component directory.
  */
-function getComponentUrl(
+export function getComponentUrl(
```

Comment:
Lets move the helper functions  `url.ts` file and export from there.. Also rename the `url.ts` file to `utils.ts`